pages/1.10/security/ent/perms-reference/index.md

```diff
@@ -33,6 +33,12 @@ By convention, `full` indicates that the permission supports all other action id
 
 ## <a name="admin-router"></a>Admin Router Permissions
 
+In each of these cases the principal is the client performing the HTTP
+request. The principal is authenticated using the DC/OS Authentication Token it
+presents in the `Authorization` HTTP header. Once authenticated, the Admin Router
```

Comment:
We seem to mostly say "Admin Router" rather than "the Admin Router"

cc @jgehrcke @vespian

Reply:
Agree.

Reply:
Fixed :+1: